aws_get.py

```diff
@@ -56,30 +35,28 @@ def stream_aws_get(scroll_id):
         aws_get_cmd = 'aws s3 sync {} {}\n'
 
         while True:
-            r = requests.post('%s/_search/scroll?scroll=10m' %
-                              es_url, data=scroll_id)
-            res = r.json()
-            logger.debug("res: %s" % pformat(res))
-            scroll_id = res['_scroll_id']
-            if len(res['hits']['hits']) == 0:
+            if len(paged_result['hits']['hits']) == 0:
```

Comment:
@mcayanan don't forget to clear the `scroll_id` whenever we use the scroll API

https://github.com/hysds/hysds_commons/blob/develop/hysds_commons/elasticsearch_utils.py#L119
```
self.es.clear_scroll(scroll_id=sid)
```

Reply:
@DustinKLo Ahh yes. Good catch. Forgot about that. Should we clear before we get the next set of results or just clear at the end? `elasticsearch_utils.py` looks like it clears at the end, but it seems like it should be before we scroll for the next set of paged results since the `clear_scroll` method takes in a `scroll_id` argument?

Reply:
Seems like we would need to pass in a list of scroll_ids used during the run? https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch.clear_scroll

Reply:
@DustinKLo does that mean we have to collect the list of scroll ids? Or just clear the current scroll id within the current loop iteration?

Reply:
According to my code in `hysds_commons` we re-use the scroll id (`sid`) and clear it at the end
https://github.com/hysds/hysds_commons/blob/develop/hysds_commons/elasticsearch_utils.py#L98-L130

Reply:
hmmm....I thought everytime we scroll, we get a new scroll_id that represents each paged result? At least that's what the code is doing on the nisar-pcm side: https://github.jpl.nasa.gov/IEMS-SDS/nisar-pcm/blob/8fe560325abbe63c43c6ddf4a3cd81880f042de2/report/get_isl_report.py#L114-L115 Do we need to update this to match what's in the `elasticsearch_utils.py`?

Reply:
i think the `_scroll_id` value returned is the same but i'll have to check first and get back to you

Reply:
For 7.1 (https://www.elastic.co/guide/en/elasticsearch/reference/7.x/paginate-search-results.html#scroll-search-results) the scroll can stay the same but is not guaranteed: 
```
The initial search request and each subsequent scroll request each return a _scroll_id. While the _scroll_id may change between requests, it doesn’t always change — in any case, only the most recently received _scroll_id should be used.
```

With that then, regarding clearing the scrolls, it seems we should accumulate them in a python `set()` then at the end clear them. What do you guys think?

Reply:
Sounds good just to be safe, i'll make the change to `ElasticsearchUtility` in `hysds_commons`
i think we should also make the change in `nisar-pcm` as well
https://github.jpl.nasa.gov/IEMS-SDS/nisar-pcm/blob/8fe560325abbe63c43c6ddf4a3cd81880f042de2/report/get_isl_report.py#L114-L115

Reply:
@pymonger @DustinKLo I've committed recent changes to clear the scrolls per this discussion: 643017d

Regarding the nisar-pcm repo, I agree that we should change it there as well.